fontc/src/change_detector.rs

```diff
@@ -9,7 +9,7 @@ use fontbe::{
 };
 use fontra2fontir::source::FontraIrSource;
 
-use crate::{create_timer, timing::JobTimer, work::AnyWork, workload::Workload, Config, Error};
```

Comment:
is there any value in keeping this file around at all? Or is this a first step?

Reply:
Probably not but I haven't got that far yet. Still WIP.

Reply:
It's gone